config/test/dsa-prototype-ec2.yaml

```diff
@@ -0,0 +1,19 @@
+template_path: "remote/basic-ec2.j2"
+stack_name: "dsa-prototype-ec2"
+dependencies:
+  - "dev/htan-vpc.yaml"
+
+parameters:
```

Comment:
default instance size is `t3.nano`.  hope that's not too small for you.

Reply:
The instance type is specified on line 7: `InstanceType: "i3en.2xlarge"`. So, this shouldn't be an issue.